example/screenshot.dart

```diff
@@ -2,60 +2,72 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-// Capture a screenshot.
-// Example comes from:
-//   https://docs.microsoft.com/en-us/windows/win32/gdi/capturing-an-image
+// Capture a multiple display screenshots.
 
 import 'dart:ffi';
-import 'dart:io';
-import 'package:ffi/ffi.dart';
 
+import 'package:ffi/ffi.dart';
 import 'package:win32/win32.dart';
 
-final hInstance = GetModuleHandle(nullptr);
-
-void captureImage(int hwnd) {
-  final hdcScreen = GetDC(NULL);
-  final hdcWindow = GetDC(hwnd);
+void main() {
+  for (final display in Displays.all()) {
+    if (display.isTurnOn) {
+      display.saveScreenshot("${display.name}.bmp");
+    }
+  }
+}
 
-  final hdcMemDC = CreateCompatibleDC(hdcWindow);
-  final bmpScreen = calloc<BITMAP>();
+class Displays {
+  static Iterable<Display> all() sync* {
+    final device = calloc<DISPLAY_DEVICE>()..ref.cb = sizeOf<DISPLAY_DEVICE>();
+    var deviceIndex = 0;
 
-  try {
-    if (hdcMemDC == 0) {
-      MessageBox(
-          hwnd, TEXT('CreateCompatibleDC failed.'), TEXT('Failed'), MB_OK);
-      return;
+    while (EnumDisplayDevices(nullptr, deviceIndex, device.cast(), 0) != 0) {
+      final isTurnOn = device.ref.StateFlags != 0;
+      yield Display(device.ref.DeviceName, isTurnOn);
+      deviceIndex++;
     }
+  }
+}
 
-    final rcClient = calloc<RECT>();
-    GetClientRect(hwnd, rcClient);
+class Display {
+  final String rawName;
+  final bool isTurnOn;
```

Comment:
Consider the name `enabled` instead:

```suggestion
  final bool enabled;
```

Reply:
For me, `isTurnOn` sounds more expressive.
It might be better to call `isConnected` as @tbsmanish [asked](https://github.com/orgs/dart-windows/discussions/603#discussioncomment-4234032)
![image](https://github.com/dart-windows/win32/assets/8049534/aac6f996-b262-480a-bee0-57b9a3800ec0)